hic_qc.py

```diff
@@ -29,12 +29,16 @@
 __version__ = get_versions()['version']
 
 # default QC thresholds if there is no thresholds file
-DEFAULT_MIN_LONG_CONTACT_PERCENTAGE     = 0.025
-DEFAULT_MIN_USEFUL_CONTACT_PERCENTAGE   = 0.025
-DEFAULT_MIN_SAME_STRAND_PERCENTAGE      = 0.05
-DEFAULT_MAX_DUPE_PERCENTAGE             = 0.4
-DEFAULT_MAX_ZERO_MAPQ_PERCENTAGE        = 0.2
-DEFAULT_MAX_UNMAPPED_PERCENTAGE         = 0.1
+DEFAULT_MIN_SAME_STRAND_HQ_PERCENTAGE           =   0.05
+DEFAULT_MIN_INFORMATIVE_READ_PAIRS_PERCENTAGE   =   0.05
+DEFAULT_MAX_NONINFORMATIVE_READ_PAIR_PERCENTAGE =   0.50
+DEFAULT_MIN_LONG_CONTACT_PERCENTAGE             =   0.025
+DEFAULT_MIN_INTERCONTIG_CONTACT_PERCENTAGE      =   0.025
+DEFAULT_MIN_USABLE_READS_PER_CONTIG             =   600
+DEFAULT_MAX_DUPE_PERCENTAGE                     =   0.40
+DEFAULT_MAX_ZERO_DIST_PERCENTAGE                =   0.20
+DEFAULT_MAX_ZERO_MAPQ0_PERCENTAGE               =   0.20
+DEFAULT_MAX_UNMAPPED_PERCENTAGE                 =   0.10
```

Comment:
Do we need these default values, or can we assume that it will always be able to access `collateral/thresholds.json`?

Reply:
The unit tests work without loading a thresholds json file. Other programmatic access could to in theory, although the commmand line interface doesn't allow you to run without a thresholds file

Reply:
Seems like it would be better to avoid the duplication and always load from the JSON

Reply:
Maybe. I wouldn't consider defaults duplication, but the main reason for doing this is unit tests (it's good to be able to test without having to use an extra library and an extra JSON file), but the unit tests could also set the thresholds to what they want to use instead. But I'll just check this in as-is since it works, is reasonable, and has the benefit of existing already.